recipe/meta.yaml

```diff
@@ -39,21 +38,19 @@ requirements:
     - m2-base                      # [win]
   host:
     - python
-    - conda-package-handling >=1.3.0
-    - menuinst >=1.4.11,<2         # [win]
-    - packaging
     - pip
-    - ruamel.yaml >=0.11.14,<0.18
-    - setuptools >=31.0.1
-    - toolz >=0.8.1
-    - pluggy >=1.0.0
-    - tqdm >=4
-    - boltons >=23.0.0
+    - hatchling 1.12.2
+    - hatch-vcs 0.2.0
+    - wheel
+    # for `conda init` in build/script above
+    - ruamel.yaml 0.17.21
+    - tqdm 4.64.0
+    - menuinst 1.4.19              # [win]
```

Comment:
Why using specific versions here instead of ranges?

`hatchling`/`hatch-vcs` being a build tools like `wheel` may require updates that would make this incompatible to this conda release and necessitate a build. Not sure if `ruamel.yaml` or `tqdm` also needs a specific pinning.

Reply:
Here is why not a range in host but exact pinning https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section
You are right, ruamel.yaml, tqdm, menuinst can be without pinnings

Reply:
All ranges on pinnings are still in `run` section